src/mapGenerator/Map.cpp

```diff
@@ -26,11 +26,11 @@ Map::Map() : width(0), height(0)
 Map::Map(unsigned int width,
          unsigned int height,
          const std::string& name,
-         const std::string& author) : width(width), height(height), name(name), author(author), players(MAX_PLAYERS)
+         const std::string& author) : width(width), height(height), name(name), author(author), positions(MAX_PLAYERS)
 {
     const unsigned int size = (unsigned int)width * height;
 
-    for (int i = 0; i < MAX_PLAYERS; i++)
+    for (unsigned int i = 0; i < positions.size(); i++)
```

Comment:
You can even do this in the ctor: `positions(MAX_PLAYERS, Point<uint16_t>::Invalid())` or use an algo like `std::fill(positions.begin(), positions.end(), Point<uint16_t>::Invalid())`